browser/brave_scheme_load_browsertest.cc

```diff
@@ -59,6 +60,31 @@ class BraveSchemeLoadBrowserTest : public InProcessBrowserTest,
     return WaitForLoadStop(active_contents());
   }
 
+  // Check loading |url| in guest window is not allowed for an url.
+  void TestURLIsNotLoadedInGuestWindow(const GURL& url) {
+    Browser* guest_browser = CreateGuestBrowser();
+    TabStripModel* guest_model = guest_browser->tab_strip_model();
+
+    // Check normal & private window have one blank tab.
+    EXPECT_EQ("about:blank",
+              guest_model->GetActiveWebContents()->GetVisibleURL().spec());
+    EXPECT_EQ(1, guest_model->count());
+    EXPECT_EQ("about:blank", active_contents()->GetVisibleURL().spec());
+    EXPECT_EQ(1, browser()->tab_strip_model()->count());
+    // Unable to navigate expected url.
+    EXPECT_FALSE(
+        content::NavigateToURL(guest_model->GetActiveWebContents(), url));
+    auto* entry = guest_model->GetActiveWebContents()
+                      ->GetController()
+                      .GetLastCommittedEntry();
+    EXPECT_EQ(entry->GetPageType(), content::PageType::PAGE_TYPE_ERROR);
+    EXPECT_STREQ("about:blank",
+                 base::UTF16ToUTF8(
+                     browser()->location_bar_model()->GetFormattedFullURL())
+                     .c_str());
+    EXPECT_EQ(1, browser()->tab_strip_model()->count());
```

Comment:
ditto

Reply:
Ah, do you want to check normal window is not affected?
If so, ignore my comments.